packages/terra-card/.npmignore

```diff
@@ -0,0 +1,6 @@
+*.log
```

Comment:
It's still possible to restore your old branch at the bottom of the original pull request that's something you want to do: https://github.com/cerner/terra-core/pull/721